src/services/SiteLaunchService.ts

```diff
@@ -1,7 +1,10 @@
+const whiteListedRepos = process.env.SITE_LAUNCH_FEATURE_WHITELISTED_REPOS?.split(
+  ","
+)
+
 export const isUserUsingSiteLaunchFeature = (siteName: string): boolean => {
-  // whitelisting sites that we want to trial this feature with
-  const whiteListedRepos = process.env.SITE_LAUNCH_FEATURE_WHITELISTED_REPOS?.split(
-    ","
-  )
+  if (siteName === "storybook") {
```

Comment:
Can i understand what's this check for? so all users using site launch feature will have sitename that is storybook?

Reply:
Oh this is solely for our storybooks so that our storybooks can see the UI changes. Concretely, when I run `npm run storybook`
I can see the site launch screens:

<img width="1444" alt="Screenshot 2023-06-09 at 1 40 54 PM" src="https://github.com/isomerpages/isomercms-frontend/assets/42832651/20318d69-128d-4b1b-b190-eea480e496d4">

alternatively, all eng would need to update their FE env var to include storybook, but opting the simpler solution here and checking it inline. added a comment [here](2a0fe7a2fe8859442382332b435ba62a2e1eb00f) for clarity

Reply:
Okay got it. But what's this env var change? Is it noted in the PR description?

> update their FE env var

Reply:
it is changed with the [correct prefix](https://create-react-app.dev/docs/adding-custom-environment-variables/) to allow react to actually source it. Yes, it's already noted in the PR desc.